requirements-django.txt

```diff
@@ -1,2 +1,2 @@
 Django
-pydantic<2.0
+pydantic
```

Comment:
Not sure how this landed in `requirements-django.txt`, but given that pydantic is in `requirements.txt` I would actually remove this requirement in `requirements-django.txt`.